igcollect/unbound.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python
+#
+# igcollect - Stats for Unbound DNS cache/resolver
+#
+# Copyright (c) 2019 InnoGames GmbH
+#
+
+from __future__ import print_function
+from argparse import ArgumentParser
+from subprocess import check_output
+from time import time
+
+
+def parse_args():
+    parser = ArgumentParser()
+    parser.add_argument('--prefix', default='unbound')
+    return parser.parse_args()
+
+
+def parse_unbound_stats():
+    stats = check_output(
+        ['/usr/local/sbin/unbound-control', 'stats'],
+        universal_newlines=True,
+        close_fds=False,
+    ).split()
+
+    out = {}
+    for stat_k, stat_v in (x.split('=') for x in stats):
+        # Report only stats for all threads
+        if stat_k.startswith('total.'):
+            stat_k = stat_k.replace('total.', '').replace('.', '_')
+            out[stat_k] = stat_v
+    return out
+
+
+def main():
```

Comment:
I bet you prefer the C-style function declaration order more. But we rather prefer the order of `argparse->main->helper_functions`.